includes/Story_Post_Type.php

```diff
@@ -153,7 +153,14 @@ static function () {
 			'the_content',
 			static function ( $content ) {
 				if ( is_singular( self::POST_TYPE_SLUG ) ) {
-					remove_filter( 'the_content', 'wpautop' );
+					remove_all_filters( 'the_content' );
```

Comment:
This basically fixes #83.

Reply:
I think if do this, we should add in a our filter at the same time. Developers should have a way of filter the content.

Reply:
What about lazy loading images and source set here, would be want these?

Reply:
Sure, we can add our own filters here and add the ones we really need. We basically do the same already for `web_stories_story_head`.

Reply:
> What about lazy loading images and source set here, would be want these?

These don't really apply here.

Lazy loading is done by AMP. And `wp_make_content_images_responsive` doesn't pick up `amp-img` anyway. So it's not something that's currently working and would be a larger effort to implement that goes beyond the scope of this PR.